$MirOS: ports/databases/postgresql/patches/patch-src_interfaces_ecpg_pgtypeslib_dt_h,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
--- src/interfaces/ecpg/pgtypeslib/dt.h.orig	Sat Mar 11 05:38:16 2006
+++ src/interfaces/ecpg/pgtypeslib/dt.h	Thu Feb 22 20:25:47 2007
@@ -330,7 +330,7 @@ int			EncodeDateOnly(struct tm *, int, c
 void		GetEpochTime(struct tm *);
 int			ParseDateTime(char *, char *, char **, int *, int, int *, char **);
 int			DecodeDateTime(char **, int *, int, int *, struct tm *, fsec_t *, int *, bool);
-void		j2date(int, int *, int *, int *);
+void		j2date(int, time_t *, int *, int *);
 void		GetCurrentDateTime(struct tm *);
 int			date2j(int, int, int);
 
